### PR TITLE
Revert "Remove non inclusive words"

### PR DESCRIPTION
**Pull request by inknos**
This reverts commit d8d29a9.
This also adds a commit to replace master with primary

### CODE REVIEW

**Comment by dralley** [comment]
[Discussion](https://github.com/rpm-software-management/libcomps/pull/78#discussion_r600503501)

I think this needs to be a full revert until the downstream impacts can be determined.

**Comment by dralley** [comment]
FWIW, I do not see the offending tags being used in Centos 7, Centos 8, or Fedora repos.  If they are no longer used, I would recommend deleting support entirely, rather than renaming in-place.

**Comment by lukash** [comment]
@dralley thanks for the heads-up and the research! :slightly_smiling_face: I fully agree.

This revert doesn't seem enough, as it only changes the XML data for tests, but not the parser (the name of the tag that is read from the data). Very curious the unittests pass, it would seem the `blacklist` is not read even in the tests. But (diverging here), looking at the tens of thousands of lines of the XML test data, the net value of the tests may not be that high, which is quite in line with the overall value-per-character of this whole repository :roll_eyes:.

Given how infrequently libcomps is released, there's not necessarily a hurry to revert this, we should have some time to investigate the possibility of dropping the `blacklist`/`blocklist` altogether.

**Comment by j-mracek** [comment]
@inknos I am really sorry. I overlooked that the original patch modify a public structure and xml output format. We can only modify a code that do not change API and in this case the format of xml is unchangeable. @inknos May I ask you for a full revert of the original PR?

File: libcomps/docs/Doxyfile.user [hunk 1]
```diff
@@ -1018,7 +1018,7 @@ HHC_LOCATION           =
 
 # If the GENERATE_HTMLHELP tag is set to YES, the GENERATE_CHI flag
 # controls if a separate .chi index file is generated (YES) or that
-# it should be included in the primary .chm file (NO).
+# it should be included in the master .chm file (NO).
```
Comment (lukash) [inline libcomps/docs/Doxyfile.user:1021]: I think we can keep the "master" -> "primary" replacement...

Comment (inknos) [reply libcomps/docs/Doxyfile.user]: I agree, I added one commit to substitute master->primary

**Comment by j-mracek** [comment]
LGTM

**Pull request merged**